Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM hayd/alpine-deno:1.4.2
+RUN apk add --no-cache python py-pip git && \
+  pip install awscli
+
+WORKDIR /app
+
+COPY deps.ts .
+RUN deno cache deps.ts && \
+  deno run -A deps.ts
+COPY test_deps.ts .
+RUN deno cache test_deps.ts
+
+ADD . .
+
+ENTRYPOINT [ "/app/run-tests.sh" ]
```

Comment:
I would really like to avoid having to compile the Docker container to run the tests. It always adds 5-10 seconds whenever I wan't to run the tests. It would be great if we could `docker-compose up -d`, and `aws s3 mb` in the `make test`, and then have the user manually tear it down via `make cleanup`.

Environment variables can also be set through the makefile.